fastlane_bot/events/utils.py

```diff
@@ -1998,7 +1998,7 @@ def handle_tokens_csv(mgr, prefix_path, read_only: bool = False):
     )
 
 
-def check_and_approve_tokens(tokens: List, cfg) -> bool:
+def check_and_approve_tokens(tokens: List, cfg: Config):
```

Comment:
return type? it used to return bool, no longer?

Reply:
Wasn't used before either, so I removed it completely.